tests/test_acr_ghosting.py

```diff
@@ -33,7 +33,7 @@ def test_ghosting(self):
 
 class TestACRGhostingGE(TestACRGhostingSiemens):
     centre = [253, 256]
-    psg = 0.471
+    psg = 0.482
```

Comment:
please explain why these values had to be updated?

Reply:
The ghosting and slice position unit tests were both updated because they both utilise the `get_mask_image` function in the ACR object, where the threshold was changed from 0.05 to 0.07 in this PR. This lead to very minor changes to the numerical results of these other tasks.